etc/storage.conf

```diff
@@ -1,31 +1,137 @@
-  graphroot = "/home/theia/.local/share/containers/storage"
-  [storage.options]
-    size = ""
-    remap-uids = ""
-    remap-gids = ""
-    ignore_chown_errors = "true"
-    remap-user = ""
-    remap-group = ""
-    ostree_repo = ""
-    skip_mount_home = ""
-    mount_program = "/usr/bin/fuse-overlayfs"
-    mountopt = ""
-    [storage.options.thinpool]
-      autoextend_percent = ""
-      autoextend_threshold = ""
-      basesize = ""
-      blocksize = ""
-      directlvm_device = ""
-      directlvm_device_force = ""
-      fs = ""
-      log_level = ""
-      min_free_space = ""
-      mkfsarg = ""
-      mountopt = ""
-      use_deferred_deletion = ""
-      use_deferred_removal = ""
-      xfs_nospace_max_retries = ""
 
+# Default Storage Driver
+driver = "vfs"
+
+# Temporary storage location
+# runroot = "/var/run/containers/storage"
+runroot = "/var/tmp/theia"
+
+# Primary Read/Write location of container storage
+# graphroot = "/var/lib/containers/storage"
+graphroot = "/home/theia/.local/share/containers/storage"
+
+[storage.options]
+# Storage options to be passed to underlying storage drivers
+
+# AdditionalImageStores is used to pass paths to additional Read/Only image stores
+# Must be comma separated list.
+additionalimagestores = [
+]
+
+# Remap-UIDs/GIDs is the mapping from UIDs/GIDs as they should appear inside of
+# a container, to the UIDs/GIDs as they should appear outside of the container,
+# and the length of the range of UIDs/GIDs.  Additional mapped sets can be
+# listed and will be heeded by libraries, but there are limits to the number of
+# mappings which the kernel will allow when you later attempt to run a
+# container.
+#
+# remap-uids = 0:1668442479:65536
+# remap-gids = 0:1668442479:65536
+
+# Remap-User/Group is a user name which can be used to look up one or more UID/GID
+# ranges in the /etc/subuid or /etc/subgid file.  Mappings are set up starting
+# with an in-container ID of 0 and then a host-level ID taken from the lowest
+# range that matches the specified name, and using the length of that range.
+# Additional ranges are then assigned, using the ranges which specify the
+# lowest host-level IDs first, to the lowest not-yet-mapped in-container ID,
+# until all of the entries have been used for maps.
+#
+# remap-user = "storage"
+# remap-group = "storage"
+
+[storage.options.overlay]
+# ignore_chown_errors can be set to allow a non privileged user running with
+# a single UID within a user namespace to run containers. The user can pull
+# and use any image even those with multiple uids.  Note multiple UIDs will be
+# squashed down to the default uid in the container.  These images will have no
+# separation between the users in the container. Only supported for the overlay
+# and vfs drivers.
+#ignore_chown_errors = false
+ignore_chown_errors = "true"
+
+# Path to an helper program to use for mounting the file system instead of mounting it
+# directly.
+#mount_program = "/usr/bin/fuse-overlayfs"
```

Comment:
If we are not using fuse-overlayfs it's probably best to remove this.

Reply:
I have copied the default configuration file and then changed necessary properties. I think it's a good practice, isn't it?

Reply:
But I'm also Ok to cleanup the file and left here only few lines.

Reply:
Let's keep it clean just for sake of consistency and making it easier to read.